deny.toml

```diff
@@ -0,0 +1,203 @@
+# This template contains all of the possible sections and their default values
```

Comment:
Is this something we should add to our ci?

At least it could be add to shell.nix. cargo-deny is packaged.

Reply:
both done. 731268e08810714c79cfece69d80a2424dd38584